mlflow/utils/validation.py

```diff
@@ -466,6 +466,11 @@ def _validate_model_name(model_name):
         raise MlflowException(missing_value("name"), error_code=INVALID_PARAMETER_VALUE)
 
 
+def _validate_model_new_name(model_name):
```

Comment:
perhaps `_validate_model_renaming` just to make it clear which API's behavior we're validating here? Other than that, this seems good to me!